awesome-tray.el

```diff
@@ -703,26 +704,36 @@ These goes before those shown in their full names."
             (format-mode-line "%p")
             )))
 
-(defun awesome-tray-module-mpc-info ()
-  (if (executable-find "mpc")
-      (let ((current-seconds (awesome-tray-current-seconds)))
-        (if (> (- current-seconds awesome-tray-mpc-command-last-time) awesome-tray-mpc-update-duration)
-            (progn
-              (setq awesome-tray-mpc-command-last-time current-seconds)
-              (awesome-tray-mpc-command-update-cache))
-          awesome-tray-mpc-command-cache))
+(with-eval-after-load 'libmpdel
+  (add-hook 'libmpdel-current-playlist-changed-hook 'awesome-tray-mpd-command-update-cache)
+  (add-hook 'libmpdel-current-song-changed-hook 'awesome-tray-mpd-command-update-cache))
+
+(defun awesome-tray-module-mpd-info ()
+  (if (executable-find "mpd")
```

Comment:
Use (if (and (ignore-errors (require 'libmpdel)) (executable-find "mpd") ...) instead.

We should use `ignore-errors (require 'xxx)``` check third-part library is installed dynamically.